CMakeLists.txt

```diff
@@ -29,30 +29,30 @@ include(CheckSymbolExists)
 # find libibverbs
 find_library(IBVERBS ibverbs)
 if (IBVERBS)
-  message(STATUS "libibverbs: ${IBVERBS}")
-  check_include_files(infiniband/verbs.h HAVE_INFINIBAND_VERBS_H)
-
-  list(APPEND CMAKE_REQUIRED_LIBRARIES -libverbs) # CMP0075, requires CMake 3.12
-  check_symbol_exists(ibv_get_device_list "infiniband/verbs.h" IBV_GET_DEVICE_LIST)
-  list(REMOVE_ITEM CMAKE_REQUIRED_LIBRARIES -libverbs)
-  if (IBV_GET_DEVICE_LIST)
-    option(WITH_VERBS "Build with verbs" ON)
-    if (WITH_VERBS)
+  option(WITH_VERBS "Build with verbs" ON)
+  if (WITH_VERBS)
+    message(STATUS "libibverbs: ${IBVERBS}")
+    check_include_files(infiniband/verbs.h HAVE_INFINIBAND_VERBS_H)
+
+    list(APPEND CMAKE_REQUIRED_LIBRARIES -libverbs) # CMP0075, requires CMake 3.12
+    check_symbol_exists(ibv_get_device_list "infiniband/verbs.h" IBV_GET_DEVICE_LIST)
+    list(REMOVE_ITEM CMAKE_REQUIRED_LIBRARIES -libverbs)
+    if (IBV_GET_DEVICE_LIST)
       set(HAVE_ENABLE_IB 1)
+    else()
+      message(STATUS "libibverbs not found")
```

Comment:
Should probably change this to something like `libibverbs found, but missing symbol ibv_get_device_list`

Reply:
Alright, I addressed this.